pyomo/gdp/plugins/cuttingplane.py

```diff
@@ -503,8 +506,6 @@ class CuttingPlane_Transformation(Transformation):
         description="""If true, sets tee=True for every solve performed over
         "the course of the algorithm"""
     ))
-    # TODO: I'd rather just have them pass the SolverFactory, if we can make
-    # that possible...
     CONFIG.declare('solver_options', ConfigValue(
```

Comment:
Why is this a `ConfigValue` with a domain of `dict` and not a `ConfigBlock(implicit=True)`?